subt/artf_node.py

```diff
@@ -27,6 +28,26 @@
 }
 
 
+def check_results(result, result_cv):
+    ret = []
+    if len(result) == 0 or len(result_cv) == 0:
+        return ret
+    for r_cv in result_cv:
+        name_cv, score_cv, bbox = r_cv
+        bbox_x1, bbox_y1, bbox_x2, bbox_y2 = bbox
+        for r in result:
+            name, points = r
+            if name != name_cv:
+                continue
+            x = np.array([p[0] for p in points])
+            y = np.array([p[1] for p in points])
+            x_in_bbox = (x > bbox_x1) & (x < bbox_x2)  # arr of boolean values
+            y_in_bbox = (y > bbox_y1) & (y < bbox_y2)  # arr of boolean values
+            if sum(x_in_bbox & y_in_bbox) >= 1:  #at least one point is in the bbox
+                ret.append(r)
```

Comment:
This is interesting. When there are two different results reported by mdnet within a single bounding box from the CV detector, do we want to keep two results, like the code here does, or a single one?

Reply:
The results will have the same artf name. I think the correct way should be merge the results. I would prefer leave it as it is for now.

Reply:
What I sometimes see is that mdnet splits a survivor into two blobs, like one arm and the body. There is a hole between them, so the detector does to try to connect them. I would expect the tf detector to put one bounding box around both of these parts of the survivor.

This is another example where I would like to think ahead, beyond "we only report a single artifact per image now".

Reply:
Ok, i tried resolved it. [98102bd](https://github.com/robotika/osgar/pull/637/commits/98102bd608e200001f32f68e893dccd0aa4400b4)